src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -509,6 +554,21 @@ private static CompletionItem CreateResourceTypeCompletion(ResourceTypeReference
                 .WithSortText(index.ToString("x8"));
         }
 
+        private static CompletionItem CreateModulePathCompletion(string name, string path, Range replacementRange, CompletionItemKind completionItemKind, CompletionPriority priority)
+        {
+            // replace windows type path sep (\\<file>) with unix path sep (/file)
+            path = StringUtils.EscapeBicepString(path.Replace(@"\\", @"/"));
+            // we do this to stay within the string ('bleh|' instead of 'bleh'|)
+            path = path.Remove(path.Length - 1);
```

Comment:
>path = path.Remove(path.Length - 1); [](start = 12, length = 36)

I'm not following the comment. What goes outside of the string and why?

(Is this a left over from a previous commit?)

Reply:
The Cursor. I will update the comment to say that explicitly